contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -81,7 +85,7 @@ func set_l1_token_bridge{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range
     # The call is restricted to the governor.
     let (caller_address) = get_caller_address()
     let (governor_) = get_governor()
-    with_attr error_message("Called address should be {caller_address}"): 
+    with_attr error_message("Called address should be {caller_address}"):
```

Comment:
Probably should be "Caller address should be {governor_}". Not your introduction but the whitespace change highlighted this mistake.

Reply:
funny thing!